src/D2L.CodeStyle.Analyzers/Common/MutabilityInspector.cs

```diff
@@ -24,6 +24,7 @@ public sealed class MutabilityInspector {
 			"System.DateTime",
 			"System.Guid",
 			"System.Lazy",
+			"System.Net.IPAddress",

```

Comment:
There's also this guy, but don't know what `Scope` is: https://referencesource.microsoft.com/#System/net/System/Net/IPAddress.cs,412